addon-test-support/@ember/test-helpers/-internal/is-component.ts

```diff
@@ -0,0 +1,25 @@
+import { macroCondition, dependencySatisfies } from '@embroider/macros';
+
+import getComponentManager from './get-component-manager';
+
+/**
+ *
+ * @private
+ * @param {Object} maybeComponent The thing you think might be a component
+ * @param {Object} owner Owner, we need this for old versions of getComponentManager
+ * @returns {boolean} True if it's a component, false if not
+ */
```

Comment:
Can you update the docs here to mention that we should ultimately be getting an API from glimmer/runtime (and link the RFC)